lib/engines/bh.js

```diff
@@ -7,37 +7,49 @@ module.exports = function (opts) {
 
     /**
      * bh.js express view engine adapter
-     * @api
-     * @returns {function(string, Object, Function)}
+     * Produces html from bh-templates and data and invokes passed callback with result
+     *
+     * @param   {String}     name       bundle name
```

Comment:
I'm not sure that this alignment is a good idea ;-( I had bad experience with that when diffs grows permanently because of long names, etc.